jvb/src/main/kotlin/org/jitsi/videobridge/EndpointK.kt

```diff
@@ -58,14 +63,47 @@ class EndpointK @JvmOverloads constructor(
     clock: Clock = Clock.systemUTC()
 ) : Endpoint(id, conference, parentLogger, iceControlling, clock) {
 
+    // TODO: this naming is to avoid conflicts with getTransceiver in Endpoint.  It will change back
+    // once Endpoint.java goes away
+    val _transceiver = Transceiver(
+        id,
+        TaskPools.CPU_POOL,
+        TaskPools.CPU_POOL,
+        TaskPools.SCHEDULED_POOL,
+        diagnosticContext,
+        logger,
+        TransceiverEventHandlerImpl(),
```

Comment:
`Endpoint` used to directly hold a reference to the `TransceiverEventHandlerImpl` instance - you decided this was not needed?

Reply:
It wasn't referenced anywhere else so I just got rid of the reference.  I don't think there's any concern about Endpoint not holding a strong reference there (Transceiver holds onto that reference, and Endpoint holds the Transceiver reference throughout its lifetime).